fs/littlefs/lfs_vfs.c

```diff
@@ -335,7 +335,7 @@ static int littlefs_close(FAR struct file *filep)
 
   if (--priv->refs <= 0)
     {
-      lfs_file_close(&fs->lfs, &priv->file);
+      ret = lfs_file_close(&fs->lfs, &priv->file);
```

Comment:
Although this may not be critical, we need to convert the return value from the `lfs_err_t` to standard `errno` value.
Most of the error codes from LittleFS map 1-to-1, except `LFS_ERR_CORRUPT`.
https://github.com/littlefs-project/littlefs/blob/master/lfs.h#L75

Reply:
Yes, but it's better to belong another patch.

Reply:
Ok